packages/runtime/src/logicsig.ts

```diff
@@ -0,0 +1,186 @@
+import {
+  decodeAddress, encodeAddress,
+  generateAccount, MultiSig, MultiSigAccount,
+  multisigAddress, signBytes, verifyBytes
+} from "algosdk";
+import * as tweet from "tweetnacl-ts";
+
+import { compareArray } from "../src/lib/compare";
+import { convertToString, stringToBytes } from "../src/lib/parsing";
+import { TealError } from "./errors/errors";
+import { ERRORS } from "./errors/errors-list";
+
+/**
+ * Note: We cannot use algosdk LogicSig class here,
+ * because algosdk.makeLogicSig() takes compiled bytes as a argument
+ * and currently we don't calculate compiled bytes from TEAL program.
+ * We are using raw TEAL code as program.(by converting string into bytes)
+ */
+export class LogicSig {
+  logic: Uint8Array;
+  args: Uint8Array[];
+  sig: Uint8Array;
+  msig: MultiSig | undefined;
+  SignatureAddress: string;
```

Comment:
Should we start class property with caps?  I think using `address` here should be ok. eg. `new LogicSig(..).address` (LogicSig already represents 'Logic Signature', so i don't think we should add Signature prefix. right?

Reply:
agreed

Reply:
but can't update to address because we have address(), updating to lsigAddress